docs/docs/overview/quickstart/installation.md

```diff
@@ -5,22 +5,23 @@ title: Installation
 
 ---
 
-Getting started with Botpress is easy. We build and distribute binaries of the latest stable version and nightly builds of the Github master branch and also have a Desktop app.
+Installing Botpress is easy. We build and distribute binaries of the latest stable version (and nightly builds) of the [GitHub master branch](https://github.com/botpress/botpress/releases). There is also a Desktop app. <!-- haven't found it, can we link to it? Or is it in the release package? -->
```

Comment:
Still unhappy about this. "Getting started with Botpress" is not true at all. 
"Installing Botpress is easy." could be a solution because I could run the server pretty quickly.

Reply:
```suggestion
Getting started with Botpress is easy. We build and distribute binaries of the latest stable version and nightly builds of the Github master branch and also have a Desktop app.
```